### ISSUE TITLE
[Feature] Allow first 'original' dynamic field to be removed

### ISSUE THREAD

**Issue by kaphert** [validformbuilder/validformbuilder#99]
### Expected behavior

If you have a dynamic area and you add a few areas you should be able to remove the first area. 

### Actual behavior

It is not possible to remove the first area. 

### Which version/branch of ValidForm Builder do you use?

I use version/branch: 
The latest version created in #94

**Comment by flangfeldt** [comment]
Removal of the "original" field should only be possible if there are more than 1 field. Also the  "remove" label should be visible only if there are more than 1 field. Otherwise it would be possible to remove a field entirely.

**Comment by kaphert** [comment]
Yes indeed, that is what I meant. 

Another option is to always show the remove label, even when there is only one left (the so called original). But instead of removing the original when there is only one left it clears all form field(s) in that dynamic area.

**Comment by flangfeldt** [comment]
I don't think that we want to have button that does two things depending on the situation, but looks the same regardless. That's just bad UX.

**Comment by rvanbaalen** [comment]
I agree. Also, a lot of the dynamic field functionality depends on the original field. Therefore it has been decided to postpone this particular feature (remove the first field) for now and wrap up the feature branch first, which is ready for review. In the feature branch, fields that are duplicated can be removed but not the original. 

---
This email was sent from my iPhone and therefore subject to typos and other inaccuracies.

**Comment by kaphert** [comment]
@flangfeldt fair point, skip that idea.
